apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Preview.tsx

```diff
@@ -62,15 +74,32 @@ export default function Preview({
       })
   }, [metadata, parameters])
 
+  useEffect(() => {
+    if (!provider) return
+    if (!conversation) return
+
+    const { messages, ruleMessage } = applyCustomRules({
+      providerType: provider.provider,
+      messages: conversation.messages,
+    })
+    setFixedMessages(messages)
+    setWarningMessage(ruleMessage)
+  }, [provider, conversation])
+
   return (
     <div className='flex flex-col flex-1 gap-2 h-full overflow-hidden'>
+      {warningMessage && (
+        <div className='w-full px-4 py-2 rounded-md bg-yellow'>
+          <Text.H6 color='warningMutedForeground'>{warningMessage}</Text.H6>
+        </div>
```

Comment:
You could add this variant `warning` to the existing Alert
https://github.com/latitude-dev/latitude-llm/blob/main/packages/web-ui/src/ds/atoms/Alert/Primitives/index.tsx#L9-L17

Reply:
Let's try not to make add-hoc components that are part of the design system.